tox.ini

```diff
@@ -21,6 +23,8 @@ deps =
     django21:  django>=2.1,<2.2
     django22:  django>=2.2,<2.3
     django30:  django>=3.0,<3.1
+    django32:  django>=3.2,<4.0
+    django40:  django>=4.0,<5.0
```

Comment:
The upper bound doesn't seem right. Expecting `<3.3` and `<4.1` here. Otherwise these targets are "moving too much" over time. Thanks!

Reply:
Fixed

Reply:
Thank you!